debian/postinst

```diff
@@ -21,13 +21,13 @@ case "$1" in
 
   configure)
 
-	chmod +x /usr/lib/zabbix-check/bin/zabbix-check-spamassassin
+    chmod +x /usr/lib/zabbix-check/bin/zabbix-check-spamassassin
 
-	if [ -x "`which invoke-rc.d 2>/dev/null`" ]; then
-		invoke-rc.d zabbix-agent restart || true
-	else
-		/etc/init.d/zabbix-agent restart || true
-	fi	
+    if [ -x "`which invoke-rc.d 2>/dev/null`" ]; then
```

Comment:
* Replace `which` with `command -v`, since `which` is not POSIX standart and maintscripts are POSIX shell scripts.
* Also a condition for systemd is missing.